src/nvim/api/buffer.c

```diff
@@ -953,6 +953,86 @@ Boolean nvim_buf_is_loaded(Buffer buffer)
   return buf && buf->b_ml.ml_mfp != NULL;
 }
 
+
+/// Unloads the buffer. See |:bunload|
+///
+/// @param buffer Buffer handle, or 0 for current buffer
+/// @param force  See <bang> description for |:bunload|
+void nvim_buf_unload(Buffer buffer, Boolean force, Error *err)
```

Comment:
do we really want three API functions that do almost the same thing? Can we not just a flag dict already